docs/providers/aws/cli-reference/login.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
Thanks! Just noticed the we should add it to the TOC as well. I'll update it ASAP 👍